massa-hash/src/hash.rs

```diff
@@ -24,6 +51,11 @@ impl PartialOrd for Hash {
     }
 }
 
+/// This function is generally useful for data structures that performs ordering and where hashes are used
+/// as keys. In massa, it is used for the BTreeMap where the order of the addresses is to be maintained.
+/// This function helps to have a single coherent BTreeMap which is then used to perform the draw
+/// See Pos-Worker for more details.
```

Comment:
What I was trying to say is that we have other use cases in Massa itself than just addresses, like block ids, operation ids, basically anything that has an underlying hash representation.

Reply:
Ok, I am gonna edit it accordingly.